ChatQnA/Dockerfile

```diff
@@ -1,8 +1,9 @@
 # Copyright (C) 2024 Intel Corporation
 # SPDX-License-Identifier: Apache-2.0
 
+ARG IMAGE_REPO=opea
 ARG BASE_TAG=latest
-FROM opea/comps-base:$BASE_TAG
+FROM $IMAGE_REPO/comps-base:$BASE_TAG
```

Comment:
Are these updated one-by-one to get that change easier though CI?
```
$ git grep -h ^FROM.*/comps-base | sort | uniq -c
      2 FROM $IMAGE_REPO/comps-base:$BASE_TAG
     13 FROM opea/comps-base:$BASE_TAG
```

Also, why not use the same variable name (REGISTRY) as in all shell scripts?
```
GenAIExamples$ git grep build.*/comps-base | cut -d. -f2 | cut -d_ -f1 | uniq -c
     24 sh:    docker build --no-cache -t ${REGISTRY}/comps-base:${TAG} --build-arg https ...
```

Reply:
Yes, I update one-by-one instead of once at all to make CI easy to pass, and I will do some other optimize in parallel.   
I use `IMAGE_REPO` instead of `REGISTRY` based on the definition of those 2 words, generally speaking `opea` is called repository rather than registry. And in our test, we sometimes use local registry for CD test, so this `$REGISTRY` actually including registry+repo, something like `100.88.111.222:5000/opea`. Above all, I think in the dockerfile it will be easier for user to understand by using `IMAGE_REPO`.

Reply:
The path before image name defines registry+project **both** for Dockerfile `FROM` command, and for `docker build` command cases.  Different values being used for them is OPEA CI special case, users who build the base + app images themselves, would use the same value for both `IMAGE_REPO` & `REGISTRY`, right?

Reply:
Yes, you are correct, from user perspective they use the same value for IMAGE_REPO & REGISTRY, that's the default one `opea`.